ai_database.py

```diff
@@ -12,54 +12,68 @@ def load_config():
     return config
 
 config = load_config()
+chat_history_enabled = config['chat_history']
+summary_enabled = config['summary']
+branching_enabled = config['branching']
+conversations_enabled = config['conversations']
```

Comment:
```suggestion
chat_history_enabled, summary_enabled , branching_enabled, conversations_enabled = \
    config[['chat_history', 'summary', 'branching', 'conversations']]
```

Reply:
```suggestion
chat_history_enabled, summary_enabled, branching_enabled, conversations_enabled = \
    config['chat_history'], config['summary'], config['branching'], config['conversations']


```
good catch. i had to do some bug fixing but i got it to work.